packages/react-components/react-utilities/src/compose/types.ts

```diff
@@ -39,16 +39,16 @@ type WithSlotShorthandValue<Props extends { children?: unknown }> =
  * Helper type for {@link Slot}. Takes the props we want to support for a slot and adds the ability for `children`
  * to be a render function that takes those props.
  */
-type WithSlotRenderFunction<Props extends { children?: unknown }> = Props & {
-  children?: Props['children'] | SlotRenderFunction<Props>;
+type WithSlotRenderFunction<Props> = Props & {
```

Comment:
@behowell  as the original author of these types, would you be able to take a look?

Reply:
Thanks for adding me--looks good!